packages/charts/react-charting/src/components/AreaChart/__snapshots__/AreaChart.test.tsx.snap

```diff
@@ -28,9 +28,7 @@ exports[`AreaChart - mouse events Should render callout correctly on mouseover 1
         &:focus {
           outline: none;
         }
-        {
-          overflow: auto;
-        }
```

Comment:
is this change intended

Reply:
Yes, overflow was always set to auto, which caused flickering scroll bars while resizing. To fix this, it is now set only when enableReflow is true.